src/test/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacadeTest.java

```diff
@@ -2,14 +2,23 @@
 
 import static org.easymock.EasyMock.createMock;
 import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.expectLastCall;
 import static org.easymock.EasyMock.replay;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static uk.ac.cam.cl.dtg.CustomAssertions.assertDeepEquals;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.ATTENDED_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.EVENT_DATE_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.EVENT_ID_FKEY_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.EVENT_TAGS_FIELDNAME;
+import static uk.ac.cam.cl.dtg.segue.api.Constants.SegueServerLogType;
```

Comment:
Is this a constant that doesn't follow the UC convention?

Reply:
It's an enum with a whole bunch of Log types such as ADMIN_EVENT_ATTENDANCE_RECORDED